lib/python/vdm/server/HTTPListener.py

```diff
@@ -2338,6 +2340,7 @@ def main(runner, amodule, config_dir, server):
     json_data = open(depjson).read()
     deployment = json.loads(json_data)
     Global.PATH = config_dir
+    Global.VOLT_ROOT_PATH = data_dir
```

Comment:
The name is very confusing here -- config_dir is being set to path, data_dir is being set to root etc. Call the global variables DATA_PATH and CONFIG_PATH and rename the parameters to main also accordingly.